params/config.go

```diff
@@ -483,8 +518,8 @@ func (c *ChainConfig) AvalancheRules(blockNum, blockTimestamp *big.Int) Rules {
 func (c *ChainConfig) enabledStatefulPrecompiles() []precompile.StatefulPrecompileConfig {
 	statefulPrecompileConfigs := make([]precompile.StatefulPrecompileConfig, 0)
 
-	if c.AllowListConfig.Timestamp() != nil {
```

Comment:
why we ommitted Timestamp nil check here?

Reply:
I believe this is omitted as it's assuming that we are replacing the meaning of a nil timestamp with a nil config entirely.